### ISSUE TITLE
middleware prevents browser-sync client.js to be injected

### ISSUE THREAD

**Issue by soenkekluth** [BrowserSync/browser-sync#217]
I need to process requested html files. if i do it like this client.js injection gets eaten.
- do you have a snippet for me to easily reincejt it or
- maybe injection should happen after all middleware did their job?

```
middleware: function(req, res, next) {

        var url = req.url === '/' ? ('/index.html') : req.url;
        var filename = __dirname + '/src' + url;

        if (url.indexOf('.html') > -1 && fs.existsSync(filename)) {

          var contents = fs.readFileSync(filename, {
            encoding: 'utf8'
          });
          res.end(contents);

        } else {
          next();
        }
      }
```

**Comment by altano** [comment]
I would reword the problem.  I should be able to modify the body in any middleware and something as simple as this doesn't work:

    function (req, res, next) {
            res.body = res.body + "modified";
            next();
    }

**Comment by shakyShane** [comment]
@altano - res.body does not exist - that's a property added express, and Browsersync does not use express

**Comment by kidwm** [comment]
@soenkekluth After #372 merged, now you can improving browsersync-ssi without script injection.
